docs/source/user/extensions.rst

```diff
@@ -4,135 +4,135 @@ Extensions
-and ``buildAvailable`` in ``jupyter_notebook_config.json`` (or ``.py`` equivalent)
-in any of the ``config`` locations returned by ``jupyter --paths``.
-
+The ``jupyter labextension`` command or the Extension Manager in JupyterLab
+enable you to list installed extensions, disable extensions without
+uninstalling them, and install or uninstall extensions distributed as npm
+packages. See the help with ``jupyter labextension --help``.
 
-.. code:: json
+.. note::
+   The Extension Manager and ``jupyter labextension`` identify an extension by
+   its JavaScript package name, which may be different from the name of the
+   ``pip`` or ``conda`` package used to distribute the extension.
 
-    {
-      "LabApp": {
-        "tornado_settings": {
-          "page_config_data": {
-            "buildCheck": false,
-            "buildAvailable": false,
-          }
-        }
-      }
-    }
 
+Listing installed extensions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+List installed extensions from the command line with
 
+.. code:: bash
 
-Using the Terminal
-~~~~~~~~~~~~~~~~~~~~~
+    jupyter labextension list
 
-Another way of managing your extensions is from the terminal on the server,
-using the ``jupyter labextension`` entry point. In general, a simple help text
-is available by typing ``jupyter labextension --help``.
+or see the list in the Extension Manager in JupyterLab.
 
 
-Installing Built-in Extensions
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Disabling Extensions
+^^^^^^^^^^^^^^^^^^^^
 
-You can install new extensions into the application
-using the command:
+You can disable specific JupyterLab extensions (including core
+extensions) without rebuilding the application by running the command:
 
 .. code:: bash
 
-    jupyter labextension install my-extension
+    jupyter labextension disable my-extension
+
+This will prevent the extension from activating in the browser.
 
-where ``my-extension`` is the name of a valid JupyterLab extension npm package
-on `npm <https://www.npmjs.com>`__. Use the ``my-extension@version``
-syntax to install a specific version of an extension, for example:
+You can enable an extension using the command:
 
 .. code:: bash
 
-    jupyter labextension install my-extension@1.2.3
+    jupyter labextension enable my-extension
 
-You can also install an extension that is not uploaded to npm, i.e.,
-``my-extension`` can be a local directory containing the extension, a gzipped
-tarball, or a URL to a gzipped tarball.
+.. _install_command:
 
-We encourage extension authors to add the ``jupyterlab-extension``
-GitHub topic to any repository with a JupyterLab extension to facilitate
-discovery. You can see a list of extensions by searching GitHub for the
-`jupyterlab-extension <https://github.com/search?utf8=%E2%9C%93&q=topic%3Ajupyterlab-extension&type=Repositories>`__
-topic.
+Installing Extensions with ``jupyter labextension``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Maybe move this up to either directly under the installing section, or at least prior to enable/disable. Reading through as is I could see someone getting confused when they try to `enable` an extension that they have not yet installed

Reply:
Thanks. I did this.